tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/PropertiesUtil.java

```diff
@@ -907,4 +919,134 @@ public static void saveProperties( Properties props, File bladeSettings )
         }
     }
 
+    public static List<IFile> getDefaultLanguagePropertiesFromModuleProject( IProject project )
```

Comment:
This method should be added as a function of a new interface "IResourceBundleProject",  So use 
```IResourceBundleProject resourceBundleProject = LiferayCore.create( project );```
Then IResourceBundleProject can be implemented by "LiferayGradleProject" or "FlexibleWebProject" so it is supported in both "web projects" and "bundle projects"